### ISSUE TITLE
Add feature to display welcome message with regex

### ISSUE THREAD

**Issue by RohanVB** [coala/corobo#659]
For any message such as `hello world!`, as long as it contains `hello world` should prompt the welcome message to be displayed

**Comment by sladyn98** [comment]
@RohanVB I'm not entirely sure why we would need this :P because we have mentioned in our docs that the message to be posted for help is the wonderful `hello world` :)

**Comment by RohanVB** [comment]
Just came across a few users using variations of the expression and thought it'd be useful. People usually explain that they need to type exactly `hello world` to trigger corobo (including the quotes). Hence the expression with quotes could be removed and everything else containing `hello world` could be included.

**Comment by Vamshi99** [comment]
Yes, `Hello world!` should be allowed. More than one `!` also should be allowed.

**Comment by sladyn98** [comment]
@RohanVB Oh understood :+1: 
@Vamshi99 Just let me get a few of my PR's merged and will get to discussing this :)
